packages/react/src/themes/teams/components/Header/headerDescriptionStyles.ts

```diff
@@ -6,12 +6,15 @@ import { HeaderDescriptionVariables } from './headerDescriptionVariables'
 import { pxToRem } from '../../../../lib'
 
 const headerStyles: ComponentSlotStylesInput<HeaderDescriptionProps, HeaderDescriptionVariables> = {
-  root: ({ props: p, variables: v }): ICSSInJSStyle => ({
-    display: 'block',
-    color: _.get(v.colors, p.color, v.color),
-    fontSize: pxToRem(22),
-    fontWeight: 400,
-  }),
+  root: ({ props: p, variables: v }): ICSSInJSStyle => {
+    const colors = v.colorScheme[p.color]
```

Comment:
```suggestion
    const colors = v.colorScheme[getColorSchemeKey(p.color)]
```

?

Reply:
We don't want that logic here, as this component implements fallback variable. We want if this is actually null to use the variable instead. I am open for other suggestions though...

Reply:
Let's address them in feature and may be we will decide to prefer schemes instead of component's fallback values 🚶

Reply:
Agreed!